racket/src/cs/core/memory.ss

```diff
@@ -40,7 +40,8 @@
   (case-lambda
    [() (collect-garbage 'major)]
    [(request)
-    (case request
+    (void)
+    #;(case request
```

Comment:
Don't forget to revert this. Does the futures implementation currently need GC to be disabled?

Reply:
Yes, Before we decided to move futures from the chez layer to the thread layer I had been working on fixing this.  I had added code that would rendezvous the threads created to run futures, so the main os thread could collect garbage, in the case that collect-garbage was called by an os thread besides the main os thread.

We had discusses before that this would be part of a separate pull request, so I was planning on adding that back after this pull request was merged.  I can revert it and then change it again in the future if you'd prefer.